test/fixtures/get-orderbook-response.json

```diff
@@ -184,13 +204,13 @@
         "makerExchangeRate": "0.003222279177208227"
       },
       "state": {
-        "availableQuantity": {
-          "currency": "USD",
+        "fundedAmount": {
```

Comment:
Yeah, `fundedAmount` is the "quantity available" for sell orders and "funds available" for buy orders.